### ISSUE TITLE
Gmail notifications are crashing launcher with "Unread notifications" enabled in At a Glance widget

### ISSUE THREAD

**Issue by th3an7** [Deletescape-Media/Lawnchair#1623]
## Description
As in the title - with Unread notifications option enabled in At a Glance widget, gmail notifs are crashing when they are cleared (for example by archiving)
Happened 2 times in a row now so I think it's actually a bug

## Expected Behavior

Clearing gmail notifications with ARCHIVE button inside it should remove info from At a Glance widget without crashing launcher

## Actual Behavior

Launcher crashes when using ARCHIVE option inside gmail notification

## Steps to Reproduce

1. Enable `Unread notifications` event source in At a Glance widget options
2. Get Gmail notification
3. Tap `Archive` in Gmail notification
4. Wait until notification disappears by itself (don't swipe it when it shows `UNDO`)

## Environment

* Device: SONY Xperia X (F5121)
* Android version: 9.0
* Launcher version: 2.0-2238-ci-alpha (2238)
* Rom: Custom ROM - AOSPA

## Logcat

 

```
error: Uncaught exception

--------- beginning of stacktrace
java.lang.IllegalStateException: sbn.notification.contentIntent must not be null
	at ch.deletescape.lawnchair.smartspace.LawnchairSmartspaceController$NotificationClickListener.<init>(LawnchairSmartspaceController.kt:442)
	at ch.deletescape.lawnchair.smartspace.NotificationUnreadProvider.getEventCard(NotificationUnreadProvider.kt:116)
	at ch.deletescape.lawnchair.smartspace.NotificationUnreadProvider.onNotificationsChanged(NotificationUnreadProvider.kt:67)
	at ch.deletescape.lawnchair.smartspace.NotificationsManager$onChange$1.invoke(NotificationsManager.kt:116)
	at ch.deletescape.lawnchair.smartspace.NotificationsManager$onChange$1.invoke(NotificationsManager.kt:34)
	at ch.deletescape.lawnchair.LawnchairUtilsKt$sam$java_lang_Runnable$0.run(Unknown Source:2)
	at android.os.Handler.handleCallback(Handler.java:873)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6697)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```
Log taken from - https://del.dog/yudevohuta.sql

**Comment by nyancrimew** [comment]
This should be fixed now

**Comment by th3an7** [comment]
Can confirm - issue fixed in 2.0-2268-ci-alpha
